solr/core/src/java/org/apache/solr/search/DocSet.java

```diff
@@ -117,11 +118,11 @@ public int andNotSize(DocSet other) {
   }
 
   /**
-   * Returns a Filter for use in Lucene search methods, assuming this DocSet
+   * Returns a Query for use in Lucene search methods, assuming this DocSet
```

Comment:
Say it runs a constant scoring Query in particular.  And I'd further remind the reader that DocSets and thus this query do not match deleted docs.

Reply:
Thanks, I'll update the documentation